src/components/include/application_manager/application_manager.h

```diff
@@ -822,6 +822,13 @@ class ApplicationManager {
    */
   virtual void OnAppUnauthorized(const uint32_t& app_id) = 0;
 
+  /**
+   * @brief ActivatePostponedApp starts postponed activation
+   * @param app application to activate
+   * @param corr_id correlation_id of the SDLActivateApp request
+   */
+  virtual void ActivatePostponedApp(uint32_t app_id, uint32_t corr_id) = 0;
```

Comment:
@LitvinenkoIra I think you can remove this function at all. Just use `GetPolicyHandler()` from state controller

Reply:
@AKalinich-Luxoft Done in a2c36cca2cc141e283b4c2fe3444cf724eb4f637